site/_components/item-grid.njk

```diff
@@ -13,8 +11,11 @@
     @param {string} modifiers.heading (optional): Designate the section heading typography class. Defaults to h1 treatment.
     @param {string} modifiers.subtitle (optional): Text for a subtitle beneath the primary title.
 
-    @param {string} locale: Code for the current locale, used when 'modifiers.variant' is 'subnav'
+    @param {object} modifiers.cta: An object with the following attributes. Locale is required when a cta is provided.
```

Comment:
It looks like `item-grid` stuff snuck in here even though this is no longer using `item-grid`. Can you please remove `item-grid` updates in this PR (njk and scss please)